docs/best-practices/xcodebuild-formatters.md

```diff
@@ -15,7 +15,7 @@ There are two main formatters the community is using. These are:
 
 As of _fastlane_ 2.201.0, `scan`, `gym`, and `snapshot` all offer a new `xcodebuild_formatter` option.
 
-This option will default to `xcbeautify` if its installed, otherwise it will fallback to `xcpretty`. However, `scan`, `gym`, and `snapshot` will end up using `xcpretty` if any options are set that are customizing with `xcpretty` (ex: `xcpretty_args`)
+"This option will default to xcbeautify if it's installed; otherwise, it will fallback to xcpretty. However, scan, gym, and snapshot will end up using xcpretty if any options are set that customize it with xcpretty (e.g., xcpretty_args)."
```

Comment:
Thanks for fixing those typos and weird wording 🤓 

What do you think about these tweaks?

```suggestion
This option will default to `xcbeautify` if it's installed; otherwise, it will fallback to `xcpretty`. However, _scan_, _gym_, and _snapshot_ will end up using `xcpretty` if any options are set that customize it with `xcpretty` (e.g., `xcpretty_args`).
```

Rationale:

- No double quotes as this is not a quote from anywhere, it's just a note to the reader
- Italicize tool names to follow our [branding guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md#branding) (I realize this was wrong before cuz it was using `code format` which was also wrong). This shouldn't be our responsibility though, I think I should bring the formatters/linters from _fastlane_ repo to this repo so it also applies the same linting rules here 👀 so don't worry much about also fixing this e.g. in line 16
- Go back to using `code format` for all terms that are names of actions/code/parameters (e.g. `xcbeautify`, `xcpretty`, `xcpretty_args`) to keep consistency across the board